packages/core/src/scenes/Inspectable.ts

```diff
@@ -67,6 +69,19 @@ export interface Inspectable {
     matrix: DOMMatrix,
     context: CanvasRenderingContext2D,
   ): void;
+
+  /**
+   * Transform the absolute mouse coordinates into the scene's coordinate system.
+   *
+   * @param x - The x coordinate.
+   * @param y - The y coordinate.
+   * @param size - (Optional) The size from the shared meta settings. Used to calculate coordinate offsets
+   */
+  transformMousePosition(
+    x: number,
+    y: number,
+    size?: {width: number; height: number},
```

Comment:
There's no need to pass this value all the way from the editor. Scenes have access to the size of the project via the `this.size` property.

Reply:
Size property was removed from the interface. I applied to fix from your other comment.